seqr/views/utils/test_utils.py

```diff
@@ -655,54 +663,62 @@ def assert_airflow_calls(self, trigger_error=False, additional_tasks_check=False
             return
 
-        # update variables
-        self.assertEqual(responses.calls[offset+1].request.url, f'{MOCK_AIRFLOW_URL}/api/v1/variables/{DAG_NAME}')
-        self.assertEqual(responses.calls[offset+1].request.method, 'PATCH')
-        self.assertDictEqual(json.loads(responses.calls[offset+1].request.body), {
-            'key': DAG_NAME,
-            'value': json.dumps(dag_variables),
-        })
-
-        # get task id
-        self.assertEqual(responses.calls[offset+2].request.url, f'{dag_url}/tasks')
-        self.assertEqual(responses.calls[offset+2].request.method, 'GET')
-
-        self.assertEqual(responses.calls[offset+3].request.url, f'{dag_url}/tasks')
-        self.assertEqual(responses.calls[offset+3].request.method, 'GET')
-
-        call_cnt = call_count - 1
-        if call_count > 5:
-            self.assertEqual(responses.calls[offset+4].request.url, f'{dag_url}/tasks')
-            self.assertEqual(responses.calls[offset+4].request.method, 'GET')
+        self._assert_update_variables_airflow_calls(offset, **kwargs)
+        call_cnt = self._assert_update_check_airflow_calls(call_count, offset)
 
         # trigger dag
-        self.assertEqual(responses.calls[offset+call_cnt].request.url, f'{dag_url}/dagRuns')
+        self.assertEqual(responses.calls[offset+call_cnt].request.url, f'{self._dag_url}/dagRuns')
         self.assertEqual(responses.calls[offset+call_cnt].request.method, 'POST')
         self.assertDictEqual(json.loads(responses.calls[offset+call_cnt].request.body), {})
 
         self.mock_airflow_logger.warning.assert_not_called()
         self.mock_airflow_logger.error.assert_not_called()
 
+    def _assert_dag_running_state_calls(self, offset):
+        self.assertEqual(responses.calls[offset].request.url, f'{self._dag_url}/dagRuns')
+        self.assertEqual(responses.calls[offset].request.method, "GET")
+
+    def _assert_update_variables_airflow_calls(self, offset, **kwargs):
+        self.assertEqual(responses.calls[offset+1].request.url, f'{self.MOCK_AIRFLOW_URL}/api/v1/variables/{self.DAG_NAME}')
+        self.assertEqual(responses.calls[offset+1].request.method, 'PATCH')
+        request_body = json.loads(responses.calls[offset + 1].request.body)
+        self.assertEqual(request_body['key'], self.DAG_NAME)
+        self.assertDictEqual(
+            json.loads(request_body['value']),
+            json.loads(json.dumps({**self._dag_variables, **kwargs}))
+        )
+
+    def _assert_update_check_airflow_calls(self, call_count, offset, check_updated_path='tasks'):
+        self.assertEqual(responses.calls[offset + 2].request.url, f'{self._dag_url}/{check_updated_path}')
+        self.assertEqual(responses.calls[offset + 2].request.method, 'GET')
+
+        self.assertEqual(responses.calls[offset + 3].request.url, f'{self._dag_url}/{check_updated_path}')
+        self.assertEqual(responses.calls[offset + 3].request.method, 'GET')
+
+        call_cnt = call_count - 1
```

Comment:
this is not actually used in this helper function so instead of computing it here and returning it you can just compute it directly above